### PR TITLE
Fix bug on timezone and on missing vlalues on add operations bulk

### PR DESCRIPTION
**Pull request by GaetanMil**
Close: #203 
Close: #190

### CODE REVIEW

File: craft_ai/client.py [hunk 1]
```diff
@@ -573,6 +573,7 @@ def _add_agents_operations_bulk(self, chunked_data):
 
         responses = []
         for chunk in chunked_data:
+            print("On a un bon chunk", chunk)
```
Comment (matthieu-boussard) [inline craft_ai/client.py:576]: LOLOOLOLOLOL

File: tests/test_pandas_add_agents_operations_bulk.py [hunk 2]
```diff
@@ -1,5 +1,7 @@
-from craft_ai.pandas import CRAFTAI_PANDAS_ENABLED
+# from craft_ai.pandas import CRAFTAI_PANDAS_ENABLED
```
Comment (matthieu-boussard) [inline tests/test_pandas_add_agents_operations_bulk.py:1]: avoid commented code, please remove it

File: tests/test_pandas_add_agents_operations_bulk.py [hunk 3]
```diff
@@ -232,6 +236,55 @@ def test_add_agents_operations_bulk_group_agents(self):
...
+            self.client.create_agent(MISSING_AGENT_CONFIGURATION, self.agent_id2)
+
+        def clean_up_agent(self, aid):
+            # Makes sure that no agent with the standard ID remains
+            self.client.delete_agent(aid)
+
+        def clean_up_agents(self, aids):
+            # Makes sure that no agent with the standard ID remains
+            for aid in aids:
+                self.clean_up_agent(aid)
```
Comment (matthieu-boussard) [inline tests/test_pandas_add_agents_operations_bulk.py:263]: don't get the point of splitting this function in two

File: CHANGELOG.md [hunk 4]
```diff
@@ -9,7 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+<<<<<<< HEAD
```
Comment (matthieu-boussard) [inline CHANGELOG.md:12]: merge conflict commited

File: craft_ai/client.py [hunk 5]
```diff
@@ -731,8 +731,11 @@ def _add_agents_operations_bulk(self, chunked_data):
                             err.__str__()
                         )
                     )
+                print(json_pl)
```
Comment (matthieu-boussard) [inline craft_ai/client.py:734]: remaining prints

File: craft_ai/client.py [hunk 6]
```diff
@@ -731,8 +731,11 @@ def _add_agents_operations_bulk(self, chunked_data):
...
                 resp = self._requests_session.post(url, headers=ct_header, data=json_pl)
+                print(resp)
```
Comment (matthieu-boussard) [inline craft_ai/client.py:736]: idem

File: craft_ai/client.py [hunk 7]
```diff
@@ -731,8 +731,11 @@ def _add_agents_operations_bulk(self, chunked_data):
...
                 decoded_response = self._decode_response(resp)
+                print(decoded_response)
```
Comment (matthieu-boussard) [inline craft_ai/client.py:738]:
```suggestion
```

File: craft_ai/pandas/client.py [hunk 8]
```diff
@@ -109,19 +110,31 @@ def add_agents_operations_bulk(self, payload):
                         "agent {}, it must be tz-aware.".format(agent["id"])
                     )
 
+                agent = super(Client, self).get_agent(agent["id"])
```
Comment (matthieu-boussard) [inline craft_ai/pandas/client.py:113]:
```suggestion
                agent = super(Client, self).get_agent(agent_id)
```

File: craft_ai/pandas/client.py [hunk 9]
```diff
@@ -97,6 +97,7 @@ def add_agents_operations_bulk(self, payload):
         new_payload = []
         for agent in valid_payload:
             operations = agent["operations"]
+            agent_id = agent["id"]
```
Comment (matthieu-boussard) [inline craft_ai/pandas/client.py:100]: if you set this variable, please use it everywhere (line 110 ?)

File: tests/data/pandas_valid_data.py [hunk 10]
```diff
@@ -158,7 +158,7 @@
         [5, "Jacques"],
         [6],
         [np.nan, OPTIONAL_VALUE],
-        [8, np.nan, "+01:00"],
+        [8, None, "+01:00"],
```
[the remark this answers is not in the document]
Comment (GaetanMil) [reply tests/data/pandas_valid_data.py:161]: It seems logic, the feature is a string so I thought it was better like that, but I can revert

**Review by matthieu-boussard — APPROVED** [review]

**Pull request merged**